big_tests/tests/graphql_muc_light_SUITE.erl

```diff
@@ -294,15 +294,27 @@ end_per_group(Group, _Config) when Group =:= user;
                                    Group =:= admin_http;
                                    Group =:= admin_cli;
                                    Group =:= domain_admin ->
+    maybe_clear_db(),
     graphql_helper:clean();
 end_per_group(Group, _Config) when Group =:= user_muc_light_with_mam;
                                    Group =:= admin_muc_light_with_mam;
                                    Group =:= domain_admin_muc_light_with_mam ->
+    maybe_clear_db(),
     ensure_muc_stopped(),
     escalus_fresh:clean();
 end_per_group(_Group, _Config) ->
+    maybe_clear_db(),
     escalus_fresh:clean().
 
+maybe_clear_db() ->
+    case mongoose_helper:is_rdbms_enabled(domain_helper:host_type()) of
+        true ->
+            F = fun() -> rpc(mim(), mod_muc_light_db_rdbms, force_clear, []) end,
+            mongoose_helper:wait_until(F, ok);
```

Comment:
Hm... why do we call it with `wait_until` if this function can only return `ok`? It looks like we never wait, and you could just call `force_clear` directly.